src/wallet/wallet.cpp

```diff
@@ -3298,7 +3298,7 @@ bool CWallet::InitLoadWallet()
             CKey key;
             key.MakeNewKey(true);
             if (!walletInstance->SetHDMasterKey(key))
-                throw std::runtime_error("CWallet::GenerateNewKey(): Storing master key failed");
+                throw std::runtime_error("CWallet::InitLoadWallet(): Storing master key failed");
```

Comment:
Maybe but it's something that can be dealt with later as part of a broader refactor which I am working on. #8152